src/orange/OrangeData.hh

```diff
@@ -58,8 +59,7 @@ struct VolumeRecord
 
     logic_int max_intersections{0};
     logic_int flags{0};
-    UniverseId daughter;
-    TranslationId daughter_translation;
+    DaughterId daughter_id;
```

Comment:
This and below could probably be replaced with an `ItemMap<LocalVolumeId, DaughterId>`: since the IDs are small we could probably just store one for every volume in the unit as part of the `SimpleUnitRecord`..

Reply:
Every volume in a unit has a VolumeRecord, so that makes sense as an ItemMap. However, the vast majority of volumes will not have daughters, so it seems strange to store a DaughterId for every volume, even if it is small. Since daughters access will all be handled in one place in the SimpleUnitTracker (per your API below), the syntactic benefit doesn't seem worth it to me.

Reply:
Either way we're storing a daughter ID for every volume: here it's just embedded into the size of the volume record. The only away to avoid that extra storage is to have a map (hash table, associative vector, etc.) that stores volume ID and daughter ID. I don't have a strong opinion about whether daughter ID is part of the volume record or the unit storage, but it seems redundant to have both.

Reply:
Okay now I see what you are saying. There is no reason to store the ItemRange of Daughters on the UnitRecord. I only did it that way to mirror what is done with the other data within UnitInserter. I fixed it so that now DaughterId is only stored on VolumeRecord.